src/me/mrCookieSlime/Slimefun/Setup/SlimefunLocalization.java

```diff
@@ -72,6 +72,7 @@ public SlimefunLocalization(SlimefunPlugin plugin) {
 		setDefaultMessage("messages.fire-extinguish", "&7You have extinguished yourself");
 		setDefaultMessage("messages.cannot-place" ,"&cYou cannot place that block there!");
 		setDefaultMessage("messages.no-pvp" ,"&cYou cannot pvp in here!");
+		setDefaultMessage("messages.radiation" ,"&4You have been exposed to deadly Radiation! &cGet rid of the radioactive Item or equip a full Hazmat Suit!");
```

Comment:
*equip the hazmat suit

Reply:
I think we should emphasize "full" here.
Because the suit consists of four pieces, "Hazmat Suit" is one of them, so that'd cause some confusion.

Reply:
also radiation shouldn't be capitalised

Reply:
Well, then make it "equip the full..."

Reply:
You brits with your weird way of capitalising :p
The entirety of Slimefun is written that way, so it should also stay.

Reply:
Why "the" and not "a".
It's not a specific hazmat suit they need, they just need any.

Reply:
There aren't multiple types of hazmat suit, there's _the_ hazmat suit.

Reply:
Okay fine.

Reply:
Watch #suggestions